aviary/cgcnn/data.py

```diff
@@ -100,16 +100,16 @@ def __repr__(self) -> str:
     def _get_nbr_data(
         self, crystal: Structure
     ) -> tuple[np.ndarray, np.ndarray, np.ndarray]:
-        """Get neighbours for every site.
```

Comment:
why americanize?

Reply:
Tools like [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) are great at reducing typos but complain about BE unless you setup custom config. And AE is more standard for code.

Reply:
I thought that https://github.com/codespell-project/codespell pre-commit hook you put in accepts both UK and US english?

Reply:
That's correct. codespell is biased towards reducing false positives. From their readme:

> This also means it shouldn't generate false-positives when you use a niche term it doesn't know about.

[Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) is different. It misses fewer typos but does incur false positive. And it's region specific, offering separate extensions and NPM packages for different languages and dialects.